domains/trix.json

```diff
@@ -4,6 +4,10 @@
     "email": "admin@trixsly.xyz"
   },
   "record": {
-    "CNAME": "superuser-l.github.io"
+    "CNAME": "tr1xem.github.io",
```

Comment:
I **wonder** whats wrong with the file. (You can't have CNAME with any other records.)